tiltak/src/main/kotlin/no/nav/amt/tiltak/deltaker/controllers/EndringsmeldingArrangorController.kt

```diff
@@ -0,0 +1,33 @@
+package no.nav.amt.tiltak.deltaker.controllers
+
+import no.nav.amt.tiltak.common.auth.AuthService
+import no.nav.amt.tiltak.common.auth.Issuer
+import no.nav.amt.tiltak.core.port.ArrangorAnsattTilgangService
+import no.nav.amt.tiltak.core.port.DeltakerService
+import no.nav.amt.tiltak.deltaker.service.EndringsmeldingService
+import no.nav.security.token.support.core.api.ProtectedWithClaims
+import org.springframework.web.bind.annotation.*
+import java.time.LocalDate
+import java.util.*
+
+@RestController
+@RequestMapping(value = ["/api/tiltakarrangor/endringsmelding"])
```

Comment:
Kan forenkles til `@RequestMapping("/api/tiltakarrangor/endringsmelding")`

Reply:
Et litt annet spørsmål, skal det hete `tiltakarrangor` eller `tiltakSarrangor` i pathen.
I samtaler så sier vi jo oftest tiltakSarrangor, hadde det gitt mening å bruke det i pathen også?

Reply:
Ja, det er jeg enig i. Gikk for tiltakarrangør fordi de andre pathene har dette så da burde man evt endre på alle